packages/gestalt/src/TableOfContents/TableOfContentsAnchor.js

```diff
@@ -0,0 +1,80 @@
+// @flow strict
+import { type Node } from 'react';
+import classNames from 'classnames';
+import styles from './TableOfContentsAnchor.css';
+import Box from '../Box.js';
+import Colors from '../Colors.css';
+import { useNesting } from '../contexts/NestingProvider.js';
+import Flex from '../Flex.js';
+import Layout from '../Layout.css';
+import TapArea, { type OnTapType } from '../TapArea.js';
+import Text from '../Text.js';
+import useInteractiveStates from '../utils/useInteractiveStates.js';
+
+const NESTING_MARGIN_START_MAP = {
+  '1': '12px',
+  '2': '32px',
+  '3': '52px',
+  '4': '72px',
+  '5': '92px',
+};
+
+const NESTING_TEXT_SIZE_MAP = {
+  '1': '300',
+  '2': '200',
+  '3': '200',
+  '4': '200',
+  '5': '200',
+};
+
+type Props = {|
+  label: string,
+  href: string,
+  active?: boolean,
```

Comment:
shouldn't `active` be required for proper behaviour?

Reply:
The use cases of the ToC requires only one item to be active at a time. Can making this prop required enforce that?
IMHO I don't think it can help with that. What are your thoughts?

Reply:
Although, making `active` required can help developers not to guess what is the prop for making an item "active"

Reply:
The idea is that all TOCs have a unique active item each time. Making it required indicates that it's part of the default state, there will be `false`s and `true`s but it must be implemented / managed for proper implementation. In all instances, the eng is going to set `active={isActive}` in one way or another.